README.md

```diff
@@ -1,4 +1,4 @@
-# Private-Groups Spec | v1.0.0
+# Private-Groups Spec | v2.0.0
```

Comment:
I think you're right this is major change

I think we can easily support backwards compatable (i.e. sigils) in this new version. I already changed `ssb-keyring` (I think that's the module) to support sigil or URI based queries (under the hood it just normalises all inputs to URI)

Reply:
Change requested: Make it clear that people MUST support sigil OR URI groupId in v2, but SHOULD use URI

compelling reason for this - we have a principle with encrypted messages that to reply to a message (a thread, or a document mutation) you should be able to copy the `recps` from the root message.
We could copy + update as we go I guess, but if supporting the old way costs nothing...

Reply:
i wonder if we need it to be backwards compatible though? would be nice to not have to include ssb-ref at all in tribes2